superset/security.py

```diff
@@ -27,6 +27,12 @@
 from superset.connectors.connector_registry import ConnectorRegistry
 from superset.exceptions import SupersetSecurityException
 
+from Crypto.Cipher import AES
+import base64
+
+BS = 16
+SK = b"qw34sd78fh67asb1"
```

Comment:
we should use some better variable names

Reply:
because of security ,using short named variable